model/github_app_auth.go

```diff
@@ -21,26 +21,30 @@ const (
 // FindOneGithubAppAuth finds the github app auth for the given project id
 func FindOneGithubAppAuth(projectId string) (*evergreen.GithubAppAuth, error) {
 	githubAppAuth := &evergreen.GithubAppAuth{}
-	q := db.Query(bson.M{githubAppAuthIdKey: projectId})
-	err := db.FindOneQ(GitHubAppAuthCollection, q, githubAppAuth)
+	err := db.FindOneQ(GitHubAppAuthCollection, byAppAuthID(projectId), githubAppAuth)
 	if adb.ResultsNotFound(err) {
 		return nil, nil
 	}
-	if err != nil {
-		return nil, err
-	}
-	return githubAppAuth, nil
+	return githubAppAuth, err
 }
 
-// HasGithubAppAuth checks if the github app auth for the given project id exists
-func HasGithubAppAuth(projectId string) (bool, error) {
-	var app *evergreen.GithubAppAuth
-	var err error
-	if app, err = FindOneGithubAppAuth(projectId); err != nil {
-		return false, err
+// byAppAuthID returns a query that finds a github app auth by the given identifier
+// corresponding to the project id
+func byAppAuthID(projectId string) db.Q {
```

Comment:
just a non-blocking nit, but should we call this `byGithubAppAuthID`? It works better for intellisense and keeps the naming conventions the same in the file. If you think not, you can just resolve this!

Reply:
yes, renamed it